src/screens/flows/Send/components/ReviewSend/index.tsx

```diff
@@ -192,6 +204,7 @@ const ReviewSend = ({
                   : t('reviewSend.holdToSend')
               }
               loading={loading}
+              onPress={() => {}}
```

Comment:
is this ok?

Reply:
yep, because rainbowButton has onPress as required. Maybe i can put it as optional in the props